app/controllers/adoption_applications_controller.rb

```diff
@@ -6,11 +6,11 @@ def new
   def create
     @adoption_application = AdoptionApplication.new(adoption_application_params)
     if @adoption_application.save
-      flash[:success] =
+      flash.now[:success] =
```

Comment:
This does not need to use the `.now` function since you redirect. The one below does because you render.